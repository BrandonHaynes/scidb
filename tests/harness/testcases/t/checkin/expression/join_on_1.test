--setup
create array A <a:int32>[x=0:2,3,0]
store(build(A,x+1),A)
store(build(A,x+2),B)
create array C <c:int32>[y=0:2,3,0]
store(build(C,y+10),C)

--test
cross_join(A,B,A.x,B.x)
cross_join(A,C,x,y)

--cleanup
remove(A)
remove(B)
remove(C)
