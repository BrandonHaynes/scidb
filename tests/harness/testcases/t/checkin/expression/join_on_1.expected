Query was executed successfully

{x} a
{0} 1
{1} 2
{2} 3

{x} a
{0} 2
{1} 3
{2} 4

Query was executed successfully

{y} c
{0} 10
{1} 11
{2} 12

{x} a,a
{0} 1,2
{1} 2,3
{2} 3,4

{x} a,c
{0} 1,10
{1} 2,11
{2} 3,12

Query was executed successfully

Query was executed successfully

Query was executed successfully

